Mathlib/RingTheory/DedekindDomain/LinearDisjoint.lean

```diff
@@ -119,3 +130,141 @@ theorem differentIdeal_eq_differentIdeal_mul_differentIdeal_of_isCoprime
     mul_comm] at this
 
 end FractionalIdeal
+
+theorem Submodule.traceDual_eq_span_map_traceDual_of_linearDisjoint [Module.Free A R₂]
```

Comment:
Can you check the names? These are still called `FractionalIdeal.blah` but there is no `FractionalIdeal`.